chef/cookbooks/cinder/recipes/ceph.rb

```diff
@@ -17,36 +17,58 @@
 # Recipe:: ceph
 #
 
-glance_servers = search(:node, "roles:glance-server")
-if glance_servers.length > 0
-  glance_pool = glance_servers[0][:glance][:rbd][:store_pool]
-else   
-  glance_pool = nil
-end
+ceph_clients = {}
+ceph_keyrings = {}
 
 node[:cinder][:volumes].each_with_index do |volume, volid|
-  if volume['backend_driver'] == "rbd"
+  if volume[:backend_driver] == "rbd"
```

Comment:
I gues it has to be `next unless volume[:backend_driver] == "rbd"` (without the if I had put there ;-))